<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MemSim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget"/>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QWidget" name="widget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="l1check">
           <property name="text">
            <string>L1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="anoldradio">
           <property name="text">
            <string>I a D společné</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="neldradio">
           <property name="text">
            <string>I a D odděleny</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>MissTime</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="l1miss">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Velikost</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="PowerSpinBox" name="l1size">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Asociativita</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="PowerSpinBox" name="l1assoc">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="l2check">
           <property name="text">
            <string>L2</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>MissTime</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="l2miss">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Velikost</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="PowerSpinBox" name="l2size">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Asociativita</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="PowerSpinBox" name="l2assoc">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="l3check">
           <property name="text">
            <string>L3</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>MissTime</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSpinBox" name="l3miss">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Velikost</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="PowerSpinBox" name="l3size">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Asociativita</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="PowerSpinBox" name="l3assoc">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QCheckBox" name="tlbcheck">
           <property name="text">
            <string>TLB</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>MissTime</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QSpinBox" name="tlbmiss">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Položky</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="PowerSpinBox" name="tlbentries">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Asociativita</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="PowerSpinBox" name="tlbassoc">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QCheckBox" name="pgtblcheck">
           <property name="text">
            <string>Tabulka stránek</string>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>AvgMissTime</string>
           </property>
          </widget>
         </item>
         <item row="19" column="1">
          <widget class="QSpinBox" name="pgtablemiss">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>LoadTime</string>
           </property>
          </widget>
         </item>
         <item row="22" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Uložit</string>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QSpinBox" name="swapmiss">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <widget class="QCheckBox" name="swapcheck">
           <property name="text">
            <string>Swap</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionRun_stp"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionRun">
   <property name="text">
    <string>R&amp;un</string>
   </property>
  </action>
  <action name="action_Load_File">
   <property name="text">
    <string>&amp;Load File</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionRun_stp">
   <property name="text">
    <string>Run step</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PowerSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">powerspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pgtblcheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>tlbcheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pgtblcheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>swapcheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>694</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
